src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -2010,6 +2010,23 @@ void DBusThreadObjectRcp::DeactivateEphemeralKeyModeHandler(DBusRequest &aReques
     otError error        = OT_ERROR_NONE;
     auto    threadHelper = mHost.GetThreadHelper();
 
+    if (otBorderAgentIsEphemeralKeyActive(threadHelper->GetInstance()))
+    {
+        error = OT_ERROR_INVALID_STATE;
```

Comment:
I wonder if there is a better error for this case, so we can distinguish with the border agent not started case. Maybe `OT_ERROR_ALREADY` or `OT_ERROR_BUSY`?

Reply:
separated stop and active with different error message. reopen if improper. Thanks~